modules/mongodb/mongodb.go

```diff
@@ -40,10 +44,44 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 		return nil, err
 	}
 
+	username := req.Env["MONGO_INITDB_ROOT_USERNAME"]
+	password := req.Env["MONGO_INITDB_ROOT_PASSWORD"]
+	if username != "" && password != "" {
+		return &MongoDBContainer{Container: container, username: username, password: password}, nil
+	}
 	return &MongoDBContainer{Container: container}, nil
 }
 
+// WithUsername sets the initial username to be created when the container starts
+// It is used in conjunction with WithPassword to set a username and its password.
+// It will create the specified user with superuser power.
+func WithUsername(username string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		req.Env["MONGO_INITDB_ROOT_USERNAME"] = username
+	}
+}
+
+// WithPassword sets the initial password of the user to be created when the container starts
+// It is used in conjunction with WithUsername to set a username and its password.
+// This environment variable sets the superuser password for MongoDB.
+func WithPassword(password string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		req.Env["MONGO_INITDB_ROOT_PASSWORD"] = password
+	}
+}
+
 // ConnectionString returns the connection string for the MongoDB container
```

Comment:
Could you update this method comment to reflect the username and the password?

Reply:
Updated it.